spock-spring/src/main/java/org/spockframework/spring/SpringBean.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.spring;
+
+import java.lang.annotation.*;
+
+/**
+ * Registers mock/stub/spy as a spring bean in the test context.
+ * <p>
+ * Inspired by Springs {@code @MockBean}, but adapted to Spock semantics.
+ * <p>
+ * Example: <pre class="code">
+ *   &#064;SpringBootTest // &#064;ContextConfiguration works also
+ *   class ExampleTests extends Specification {
+ *     &#064;SpringBean
+ *     Service mockService = Mock()
+ *
+ *     &#064;SpringBean
+ *     Service stubService = Stub() {
+ *       stubCall() >> "default interaction can be defined to"
+ *     }
+ *
+ *     &#064;SpringBean
+ *     Service spyService = Spy()
+ *   }
+ *   </pre>
+ * <p>
+ * Note, that like {@code @MockBean} this will modify your {@code ApplicationContext}
+ * an thus will prevent its reuse for other specs.
+ * <p>
```

Comment:
closing < p > Tag?

Reply:
Not really in Javadoc since it is based on HTML 3.2 see https://stackoverflow.com/questions/5260368/which-tag-should-be-used-as-paragraph-separator-in-javadoc